factory/factory.Dockerfile

```diff
@@ -111,6 +111,9 @@ ONBUILD RUN node /opt/installScripts/cypress/install-cypress-version.js ${CYPRES
 
 # Global Cleanup
 ONBUILD RUN apt-get purge -y --auto-remove \
+    curl \
+    libcurl4 \
+    libcurl3-gnutls \
```

Comment:
not sure if these need to be removed since i'd assume they'd be removed by purging curl, but it doesn't seem to hurt.